### PR TITLE
Move snowplow to its own script

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: portal-frontend/src/assets/snowplow.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  platform: "web",
+  post: true,
+  forceSecureTracker: true,
+  contexts: {
+    webPage: true, performanceTiming: true
+  }
+});
+window.snowplow("enableActivityTracking", 30, 30); // Ping every 30 seconds after 30 seconds
+window.snowplow("enableLinkClickTracking");
+window.snowplow("trackPageView");
```
Comment (trslater) [inline portal-frontend/src/assets/snowplow.js:30]: Doesn't the service in your previous already call `trackPageView`? Why do we need it here as well?

Comment (dhaselhan) [reply portal-frontend/src/assets/snowplow.js:30]: This was the code given to us by the snowplow team to paste into the index file. I am not sure.

Comment (dhaselhan) [reply portal-frontend/src/assets/snowplow.js]: Removed

**Review by trslater — APPROVED** [review]
LGTM 🚀

**Pull request merged**